components/ExperienceCard.tsx

```diff
@@ -0,0 +1,44 @@
+"use client"
+import React from 'react';
+import {Card, CardHeader, CardBody, Image} from "@nextui-org/react";
+
+type Props = {
+    experiences: { 
+        data: 
+        { 
+            img: string; 
+            employer: string; 
+            job_title: string; 
+            years_worked: string;
+        }[];
+    };
+}
+
+const ExperienceCard = (props: Props) => {
+  return (
+    <>
+        {props.experiences.data.map((exp, idx) => (
+            <Card key={idx} className="w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] bg-white-900 shadow-work cursor-pointer sm:w-[607px] sm:border sm:border-white-800 sm:dark:border-none sm:shadow-none sm:hover:shadow-work sm:hover:ml-[30px] dark:bg-transparent hover:dark:bg-black-300">
```

Comment:
You could consider moving some of this to a class in your globals.css

You can make a utility class to handle a lot if this:

Globals.css
```css
@layer utilities{
 .Card{
   @apply shadow-work cursor-pointer bg-white-900 etc...etc...
 }
 .Card:hover{
  @apply sm:hover:shadow-work sm:hover:ml-[30px] etc.. etc..
 }
}
```

If you find yourself using a lot of these in groups over and over- that a good way to clean it up. You can also extend your tailwind theme- But regardless I think you might have reached a limit for reasonable amount of tailwind classes in one className 😂

Reply:
Haha, I knew it was getting out of hand. I'll make custom classes as you suggested.